src/version3/issueComments.ts

```diff
@@ -145,7 +145,23 @@ export class IssueComments {
    */
   async addComment<T = Models.Comment>(parameters: Parameters.AddComment, callback?: never): Promise<T>;
   async addComment<T = Models.Comment>(parameters: Parameters.AddComment, callback?: Callback<T>): Promise<void | T> {
-    // todo add simple comment structure (string)
+    const getBody = (body: Parameters.AddComment['body']): Models.Document | undefined => {
```

Comment:
```suggestion
    const body = typeof parameters.body === 'string' ? {
      type: 'doc',
      version: 1,
      content: [
        {
          type: 'paragraph',
          content: [{ type: 'text', text: parameters.body }],
        },
      ],
    } : parameters.body;
```

Reply:
humm it does solve the error, but it looks like it simply disables that rule
https://github.com/prettier/eslint-config-prettier/blob/88ba724915c0d52c822d7c0d499be21c30c5380a/index.js#L90

I'd rather not make too big a change in your linter rules in this pull request by adding a whole other configuration.

Reply:
Yeah, okay. I will complete all work with linter by myself. Conceptually everything was done and I express my great gratitude for your contribution!